doc/en/user/source/community/ogc-api/features/index.rst

```diff
@@ -0,0 +1,202 @@
+
+* ``html``: A collection of web pages, with links for navigation between content (and that can be indexed by search engines for discoverability).
+
+  .. figure:: img/features.png
+ 
+     OGC API Features service
+
+* `application/json`: A collection :file:`json` documents, with reference between each document for programatic access by web developers.
+
+  .. code-block:: json
+  
+     {
+       "title": "GeoServer Web Feature Service",
+       "description": "This is the reference implementation of WFS 1.0.0 and WFS 1.1.0, supports all WFS operations including Transaction.",
+       "links": [
+         {
+           "href": "http://localhost:8080/geoserver/ogc/features/?f=application%2Fx-yaml",
+           "rel": "alternate",
+           "type": "application/x-yaml",
+           "title": "This document as application/x-yaml"
+         },
+         {
+           "href": "http://localhost:8080/geoserver/ogc/features/?f=application%2Fjson",
+           "rel": "self",
+           "type": "application/json",
+           "title": "This document"
+         },
+         {
+           "href": "http://localhost:8080/geoserver/ogc/features/?f=text%2Fhtml",
+           "rel": "alternate",
+           "type": "text/html",
+           "title": "This document as text/html"
+         },
+         ...
+
+* ``application/x-yaml``: A collection of :file:`yaml` documents, with references between each document for programatic access.
+ 
+  .. code-block:: yaml
+  
+     title: GeoServer Web Feature Service
+     description: This is the reference implementation of WFS 1.0.0 and WFS 1.1.0, supports
+       all WFS operations including Transaction.
+     links:
+     - href: http://localhost:8080/geoserver/ogc/features/?f=application%2Fx-yaml
+       rel: self
+       type: application/x-yaml
+       title: This document
+     - href: http://localhost:8080/geoserver/ogc/features/?f=application%2Fjson
+       rel: alternate
+       type: application/json
+       title: This document as application/json
+     - href: http://localhost:8080/geoserver/ogc/features/?f=text%2Fhtml
+       rel: alternate
+       type: text/html
+       title: This document as text/html
+
+The service title and description are provided by the existing :ref:`wfs` settings.
+
+Open API
+''''''''
+
+For programatic access an `OpenAPI <https://www.openapis.org/>`__ description of the service is provided, that may be browsed as documentation, or used to generate a client to access the web services.
+
+.. figure:: img/features-api.png
+   
+   OGC API Features OpenAPI Document
+
+Collections
+'''''''''''
+
+The collection of feature types being published by the service.
+
+Each collection entry is described using the layer details of title, description, geogrpahic extent.
+
+Data can be browsed as html pages, or downloaded in a range of formats such as :file:`GeoJSON` and :file:`GML`.
+
+.. figure:: img/collection.png
+   
+   Collection sf:roads download formats
+
+Tile matrix sets
```

Comment:
These are extension provided by the "tiled-features" module. It basically allows to download data as tiles (typically vector tiles). Once we split the ogcapi in separate downloads (I hope to do it during the sprint) this section might not show up anymore, unless people also install the tiles API)

Reply:
Okay, I will need to think about how to document extensions.